.editorconfig

```diff
@@ -0,0 +1,25 @@
+root = true
+
+[*]
+indent_style = space
```

Comment:
note sure this would apply to all files, for example the sphinx Makefile in doc.